content/helloworld-fromcftok8s.md

```diff
@@ -253,10 +255,10 @@ spec:
           servicePort: 8080
 ```
 
-Note that I'm using the xip.io service to cheat around the fact that I do not have a public DNS entry that resolves to my k3s VM's IP address. I could have entered this into the /etc/hosts file on my laptop but this way is just simpler and quicker. In case you don't know what xip.io does, it simple cuts out the IP address from the URL and sends it back as the IP address this name resolves to. Why didn't I have this idea myself ages ago? Don't say it...
+Note that I'm using the xip.io service to cheat around the fact that I do not have a public DNS entry that resolves to my K3s VM's IP address. I could have entered this into the /etc/hosts file on my laptop but this way is just simpler and quicker. In case you don't know what xip.io does, it simple cuts out the IP address from the URL and sends it back as the IP address this name resolves to. Why didn't I have this idea myself ages ago? Don't say it...
 
 And now, finally, we are back to where we started. What a journey. All just to reproduce what a single 'cf push' command does. 
 
-Is this because I'm the first one noting that there's a bit of a complexity problem here? Not at all. If there's one thing that's certain, then it is that with everything that looks like a really good idea, most of the time someone else thought about it already. Same here. There's a ton of tools out there which try to make things simpler for developers when developing apps that only run properly once deployed to Kubernetes. More on that in a future post - stay tuned. 
+Am I the first one to note that there's a bit of a complexity problem here? Not at all. If there's one thing that's certain, when something looks like a really good idea, most of the time someone else has already thought of it. Same here. There are tons of tools out there that try to make things simpler for developers when developing apps that only run properly once deployed to Kubernetes. More on that in a future post: stay tuned. 
 
-If you've made it until here, thank you very much for bearing with me. Remember, the only stupid question is the unasked one and everybody else is just googling for stuff just like you are. 
\ No newline at end of file
+If you've made it this far, thank you very much for bearing with me. Remember, the only stupid question is the unasked one. Everybody else is just Googling for stuff too. 
```

Comment:
Tim, can i get your title and bio? And what picture do you want to use? I pulled the one from Linkedin.